README.md

```diff
@@ -569,3 +569,37 @@ When you create a release branch in the form of `release/yyyy-mm-dd` or `hotfix/
 The CI pipeline will check to make sure you have done this to prevent any mistakes - if you have made a release or hotfix branch it will check that the value in the RELEASE file matches or not
 
 This is because it will use that value to tag the commit once its been merged into main as a reference point, and this is how it tracks which release it is as github actions struggles with post merge branch identification
+
```

Comment:
worth amending and duplicating parts of this with the section `4. Run the Feature Tests`

Reply:
moved into section 4